lib/src/components/NxRadio/NxRadio.scss

```diff
@@ -18,3 +18,14 @@
 .nx-radio__input:checked + .nx-radio__circle .nx-radio__outer-circle {
   fill: none;
 }
+
+.nx-radio__input:checked:disabled {
```

Comment:
Up to you, but if it were me I'd combine this block and the one above since they both start with `.nx-radio__input:checked`

Reply:
> Up to you, but if it were me I'd combine this block and the one above since they both start with `.nx-radio__input:checked`

Combined the two blocks
1dd61c9d487c481deb86b07bfd3dba9c92d7551b

PS: For some reason, the `+ .nx-radio__circle` selector couldn't properly select the element. I had to change it to `+ svg.nx-radio__circle ` to select the element so that I didn't get the blue circle on hover.

Reply:
> PS: For some reason, the + .nx-radio__circle selector couldn't properly select the element. I had to change it to + svg.nx-radio__circle to select the element so that I didn't get the blue circle on hover.

It seems to be working without the `svg` to me

Reply:
It's not quite the same for me.

With the `svg` part added, on hover it looks like:
![image](https://user-images.githubusercontent.com/7110862/123654869-7cc5cb80-d7fc-11eb-9003-dc8f9d3f9351.png)

Without the `svg` part added, on hover it looks like:
![image](https://user-images.githubusercontent.com/7110862/123654992-99fa9a00-d7fc-11eb-8fcf-da42cfdb3053.png)

I don't think the `svg` part of it _hurts_ anything as `.nx-radio__circle` is a child of the `svg` element.

Reply:
> I don't think the svg part of it hurts anything as .nx-radio__circle is a class on the svg element.

Ok yeah I hadn't noticed the hover issue.  I'm sure it's just a specificity thing.